baybe/recommenders/naive.py

```diff
@@ -25,28 +27,24 @@ class NaiveHybridRecommender(Recommender):
     a non-hybrid space, it uses the corresponding recommender.
     """
 
-    # TODO: This class (and potentially the recommender function signatures) need to
-    #   be refactored such that there is no more coupling to BayesianRecommender and it
-    #   can be moved to recommender.py
-
     # Class variables
     compatibility: ClassVar[SearchSpaceType] = SearchSpaceType.HYBRID
     # See base class.
 
     # Object variables
     # TODO This used to be a Union of BayesianRecommender and NonPredictiveRecommender.
-    # Due to serialization issues, this was changed to Recommender in general.
-    # As we currently do not have other subclasses of Recommender, this solution works
-    # for now. Still, we manually check whether the disc_recommender belogns to one of
-    # these two subclasses such that we might be able to easily spot a potential problem
-    # that might come up when implementing new subclasses of Recommender
-    disc_recommender: Recommender = field(factory=SequentialGreedyRecommender)
+    # Due to serialization issues, this was changed to PureRecommender in general.
+    # As we currently do not have other subclasses of PureRecommender, this solution
```

Comment:
This description is now longer correct since we have subclasses of `PureRecommender` now, right? So this description should be adjusted slightly.

Reply:
i dont get what you mean, i dont think the issue the text describes has changed

Reply:
I mean that we know have **other** subclasses of `PureRecommender`, so the exact wording is not correct, or am I mistaken here?

Reply:
which new ones do we have?

Reply:
Checked again, seems like I was mistaken here resp. had something wrong in mind.